CHANGELOG.md

```diff
@@ -43,6 +43,23 @@ system.io {
 }
 ```
 
+* Added a `script-epilogue` configuration option to adjust the logic that runs at the end of the scripts which wrap call executions.
+  This option is adjustable on a per-backend basis.  If unspecified, the default value is `sync`.  To turn off the default `sync` 
+  behavior on the `Local` backend:
```

Comment:
I do this a lot and tend to agree with commenters who say, it's nicer if the details are in the real docs, and the changelog just tells you where to look.

Certainly I've regretted putting details only in the changelog before (e.g. for `concurrent-job-limit`) and then struggling to find that information when I search for it later in the README...